tests/scripts/minikube.sh

```diff
@@ -101,6 +109,19 @@ case "${1:-}" in
     copy_image_to_cluster ${BUILD_REGISTRY}/rook-amd64 rook/rook:master
     copy_image_to_cluster ${BUILD_REGISTRY}/toolbox-amd64 rook/toolbox:master
     ;;
+  restart)
+    # If current kubectl context is minikube, restart rook pods
+    if [ "$(kubectl config view 2>/dev/null | awk '/current-context/ {print $NF}')" = "minikube" ];
```

Comment:
everywhere else we assumed minikube is the context. seems like we could assume it here to keep the script simple

Reply:
To be on safe side and prevent any unintended Kube cluster modifications, especially when we are deleting pods, let's leave it in place, but I'll make context check to be a separate function so we could reuse it for other intrusive actions. Updating.